kv/dist_sender.go

```diff
@@ -402,7 +402,7 @@ func (ds *DistSender) sendRPC(desc *proto.RangeDescriptor,
 	}
 
 	// Copy and rearrange the replicas suitably, then return the desired order.
-	replicas := proto.ReplicaSlice(append([]proto.Replica(nil), desc.Replicas...))
+	replicas := append(proto.ReplicaSlice{}, desc.Replicas...)
```

Comment:
Note that `proto.ReplicaSlice{}` and `proto.ReplicaSlice(nil)` are different. The former does an allocation creating a non-nil but empty slice. The latter does not.